mmv1/products/netapp/volumesnapshot.yaml

```diff
@@ -0,0 +1,83 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'volumesnapshot'
```

Comment:
Should be named VolumeSnapshot, as well as this file should be title case

Reply:
Snapshots are a nested resource of volumes. So a final name liek google_netapp_volumesnapshot sounds like the right name, doesn't it?
Here a list of other resources in the netapp space:
[
  "google_netapp_active_directory",
  "google_netapp_backup_policy",
  "google_netapp_backup_vault",
  "google_netapp_kmsconfig",
  "google_netapp_storage_pool",
  "google_netapp_volume",
  "google_netapp_volumereplication"
]
google_netapp_volumereplication is currently also under review in an PR. It uses the same name schema.

We can easily capitalise the name and the filename, but want to verify first: This doesn't have any influence on the resource name, right? They still will all be lower case, right?

Reply:
Yeah, the resource name will never have capital letters, they just signify where the underscore should go.

I would say volumereplication should also be volume_replication, or just replication.

Reply:
Ah, now I get you. Ack.
@G-NamanGupta , please change
`name: 'volumesnapshot'` to `name: 'VolumeSnapshot'` and the filename to VolumeSnapshot.yaml.
This also will have an impact on the tests, as the resource name will change from `google_netapp_volumesnapshot` to `google_netapp_volume_snapshot`.